assets/lang/pl_PL.ini

```diff
@@ -615,7 +615,7 @@ Load = Otwórz...
 Loading... = Loading...
 PinPath = Przypnij
 PPSSPP can't load games or save right now = PPSSPP nie może teraz załadować gier ani zapisywać stanów
-Recent = Ostatnio grane
+Recent = Ostatnio grywane
```

Comment:
Nie wiem w jakim zakątku Polski mówimy "grywane", trochę słabo to brzmi. Już lepiej i bardziej neutralnie byłoby "Ostatnio uruchomione".

Reply:
Generalnie to bym zamienił na "Ostatnio uruchamiane" tym bardziej, że PPSSPP uruchamia nie tylko gry ale i aplikacje, do tego większość "gier" z biblioteki PSP to powieści wizualne i wiele osób nawet ich nie zalicza do gier jako takich, analogicznie jak audiobooki nie są muzyką.

Reply:
zgadzam się, że to dość rzadko używana forma, ale jak to tłumaczyłem nie miałem lepszego pomysłu ;-)
może rzeczywiście "Ostatnio uruchamiane" byłoby lepse - zmieniam

> tym bardziej, że PPSSPP uruchamia nie tylko gry ale i aplikacje

ale z drugiej strony "Last played" -> `play` = grać/odtwarzać